lib/fdoc/spec_watcher.rb

```diff
@@ -8,7 +8,8 @@ module SpecWatcher
       define_method(verb) do |*params|
         action, request_params = params
 
-        super(*params)
+        request_params.nil? ? 
+        super(*params) : super(action, **request_params)
```

Comment:
one line is fine here for the ternary

```suggestion
        request_params.nil? ? super(*params) : super(action, **request_params)
```

I think we could also use [`ruby2_keywords`](https://github.com/ruby/ruby2_keywords) here by calling `ruby2_keywords verb` after the `define_method` (if `ruby2_keywords` is defined), but this also seems fine